rllib/policy/policy_map.py

```diff
@@ -34,29 +24,25 @@ class PolicyMap(dict):
 
     def __init__(
         self,
-        worker_index: int,
-        num_workers: int,
+        *,
         capacity: Optional[int] = None,
-        path: Optional[str] = None,
-        policy_config=None,  # deprecated arg: All Policies now bring their own config.
-        session_creator: Optional[Callable[[], "tf1.Session"]] = None,
-        seed: Optional[int] = None,
+        policies_swappable: bool = False,
+        # Deprecated args.
+        worker_index=None,
+        num_workers=None,
+        policy_config=None,
+        session_creator=None,
+        seed=None,
     ):
         """Initializes a PolicyMap instance.
 
         Args:
-            worker_index: The worker index of the RolloutWorker this map
-                resides in.
-            num_workers: The total number of remote workers in the
-                WorkerSet to which this map's RolloutWorker belongs to.
             capacity: The maximum number of policies to hold in memory.
```

Comment:
From the docs of dequeue: Unbounded! It might as well be part of the docstring. https://docs.python.org/3/library/collections.html#collections.deque.maxlen

Reply:
Good points @ArturNiederfahrenhorst and @kouroshHakha . Fixed both.